src/recording/recordingmanager.cpp

```diff
@@ -145,23 +167,37 @@ QString& RecordingManager::getRecordingDir() {
 }
 
 // Only called when recording is active.
-void RecordingManager::slotDurationRecorded(QString durationStr)
+void RecordingManager::slotDurationRecorded(quint64 duration)
```

Comment:
I was just adapting the code to what we had. Duration represents seconds, so it really doesn't need that much precision. It's just that it already was quint64 in the enginerecord class.